### PR TITLE
Update tutorial content

### PR DESCRIPTION
**Pull request by chriswalker**
Addresses #553 .

Updates tutorial text and tool output to reflect recent changes in the tooling. Primarily:

1. Update examples of `kev`'s output, mainly in the "Getting Started With Kev" tutorial
2. Consistently output terminal commands across all tutorials
3. Tweaks to markdown across tutorials for consistency and improved readability (for example, block quoting)

### CODE REVIEW

File: docs/tutorials/quickstart-guide.md [hunk 1]
```diff
@@ -100,7 +100,8 @@ To deploy your app onto Kubernetes,
 In this example, we deploy the `stage` environment:
 
 ```sh
-kubectl apply -f k8s/stage     # deploys your app with stage settings onto the default namespace
+# deploys your app with stage settings onto the default namespace
+kubectl apply -f k8s/stage
```
Comment (marcinc) [inline docs/tutorials/quickstart-guide.md:104]: minor: could prefix with a `$` as we do in other place.

**Review by marcinc — COMMENTED** [review]
Left one minor comment. LGTM otherwise.

File: docs/tutorials/quickstart-guide.md (hunk 1 above)
Comment (chriswalker) [reply docs/tutorials/quickstart-guide.md]: Oh, nuts, I _knew_ I'd miss one somewhere 🤦  - fixed, nice catch

**Review by marcinc — APPROVED** [review]
LGTM 👍

File: docs/tutorials/simple-nodejs-app-workflow.md [hunk 2]
```diff
@@ -38,9 +40,9 @@ It'll also bootstrap the Skaffold config file (`skaffold.yaml`). If skaffold.yam
 
 Once changes to base & environment specific Compose file(s) have been made
 
-> One-off Kubernetes manifests render:
 ```sh
-kev render
+# One-off Kubernetes manifests render :
```
Comment (ezodude) [inline docs/tutorials/simple-nodejs-app-workflow.md:44]: `render :` -> `render:`

File: docs/tutorials/getting-started-with-kev.md [hunk 3]
```diff
@@ -289,53 +391,106 @@ Also, we've made some minor adjustments to the `wordpress` service. Kev will rec
 
 This will be applied to all _Compose environment overrides_.
 
-Simply, re-run,
-```shell script
+We hve already initialised kev, so to catch the new changes to the Compose file we simply need to re-run,
```
Comment (ezodude) [inline docs/tutorials/getting-started-with-kev.md:394]: `hve` -> `have`

**Review by ezodude — COMMENTED** [review]
Great work! .... just a couple of minor fixes.

File: docs/tutorials/getting-started-with-kev.md (hunk 3 above)
Comment (chriswalker) [reply docs/tutorials/getting-started-with-kev.md:394]: Nice catch, fixed

**Review by ezodude — APPROVED** [review]
👍

**Pull request merged**